### PR TITLE
Documentation for 3.0.0 release

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by MichaelBailly** [comment]
Can we have screenshots of the resulting page(s) ?

File: src/homepage/_posts/2017-07-20-james-3.0.markdown [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Apache James Server 3.0.0"
+date:   2017-07-20 16:13:22 +0200
+categories: james update
+---
+
+James server 3.0.0 has been released !!!
```
Comment (MichaelBailly) [inline src/homepage/_posts/2017-07-20-james-3.0.markdown]: Why 3 "!" ?

File: src/site/xdoc/server/index.xml [hunk 2]
```diff
@@ -40,18 +40,18 @@
...
-              <span><span class="icon"></span>Early Apache James Server 3.0-RC1</span>
+              <span><span class="icon"></span>Apache James Server 3.0.0</span>
             </a>
           </span>
         </p>
          <p>You can also have a look to the last stable version</p>
 
-         <p>The James team is working hard to release version 3.0. We currently do load testing, write more tests, rework the website,
-             and started using James 3.0 in production.</p>
+         <p>The James team is working hard to release version 3.0.0. We currently do load testing, write more tests, rework the website,
```
Comment (MichaelBailly) [inline src/site/xdoc/server/index.xml]: Is that section still accurate, now that the 3.0.0 is released ?

File: src/site/xdoc/server/release-notes.xml [hunk 3]
```diff
@@ -35,6 +35,31 @@
...
+                of version 3.0.0 of its modular mail server.</p>
+
+            <p>After a long work, the team is really proud to announce the new release, James 3.0.
+                This new version of the Apache James Server introduces many new features, 
+                like the IMAP protocol, 
+                experimental Sieve support, 
+                experimental JMAP support, 
+                and new email storage back-ends.</p>
+
+            <p>For the James 3.0 release, the team have been specifically committed to testing.
```
Comment (MichaelBailly) [inline src/site/xdoc/server/release-notes.xml]: the team has been

Comment (MichaelBailly) [inline src/site/xdoc/server/release-notes.xml]: well, not sure, maybe double check with a great english speaker

**Comment by aduprat** [comment]
capture du 2017-07-20 18 03 12
capture du 2017-07-20 18 02 55
capture du 2017-07-20 18 00 10

File: src/site/xdoc/server/quick-start.xml [hunk 4]
```diff
@@ -44,16 +44,16 @@ Step 0: Requirements
...
 #################
 
-  * Download apache-james-3.0-beta5-app.zip from http://james.apache.org/download.cgi#Apache_James_Server
+  * Download apache-james-3.0.0-app.zip from http://james.apache.org/download.cgi#Apache_James_Server
 
 Step 2: Deploy
 ##############
 
-  * Unzip apache-james-3.0-beta5-app.zip.
+  * Unzip apache-james-3.0-app.zip.
```
Comment (quynhn) [inline src/site/xdoc/server/quick-start.xml]: 3.0.0?

File: src/site/xdoc/server/quick-start.xml [hunk 5]
```diff
@@ -44,16 +44,16 @@ Step 0: Requirements
...
   * You should have a folder with sub-folders bin, conf, lib, log, var and four text files.
 
-$ unzip japache-james-3.0-beta5-app.zip
-$ cd apache-james-3.0-beta5-app
+$ unzip japache-james-3.0-app.zip
```
Comment (quynhn) [inline src/site/xdoc/server/quick-start.xml]: s/japache/apache/

Comment (quynhn) [inline src/site/xdoc/server/quick-start.xml]: s/3.0/3.0.0/

File: src/site/xdoc/server/quick-start.xml [hunk 6]
```diff
@@ -44,16 +44,16 @@ Step 0: Requirements
...
+$ cd apache-james-3.0-app
```
Comment (quynhn) [inline src/site/xdoc/server/quick-start.xml]: 3.0.0

File: src/homepage/_posts/2017-07-20-james-3.0.markdown (hunk 1 above)
Comment (aduprat) [inline src/homepage/_posts/2017-07-20-james-3.0.markdown]: One per version !!!

File: src/site/xdoc/server/index.xml (hunk 2 above)
Comment (aduprat) [inline src/site/xdoc/server/index.xml]: Right -> removed

File: src/site/xdoc/server/release-notes.xml (hunk 3 above)
Comment (aduprat) [inline src/site/xdoc/server/release-notes.xml]: It seems that both are accepted...

**Pull request merged**